src/OrbitGl/TrackManagerTest.cpp

```diff
@@ -127,9 +130,28 @@ TEST_F(TrackManagerTest, EmptyTracksAreNotVisible) {
 }
 
 TEST_F(TrackManagerTest, NonEmptyTracksAreVisible) {
-  CreateAndFillSchedulerTracks();
+  CreateAndFillTracks();
```

Comment:
This line change should be in the previous commit or you should combine them together

Reply:
Nice catch, thanks, fixed (not yet pushed).